modules/evaluate_condition_abstract.py

```diff
@@ -0,0 +1,21 @@
+"""
```

Comment:
Nit: Rename the file from `evaluate_condition_abstract.py` to `evaluate_condition.py`.